### PR TITLE
Remove Zipkin Extension Documentation from Swan Lake

### PR DESCRIPTION
**Pull request by nadundesilva**
## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: swan-lake/learn/observing-ballerina-code.md [hunk 1]
```diff
@@ -37,10 +37,10 @@ Prometheus, etc. You can follow [Docker documentation](https://docs.docker.com/i
 
 ### Step 2 - Installing and Configuring the External Systems
 
-* Setup Prometheus for collecting metrics information by following section on [Setting up Prometheus](#prometheus)
-* Setup Grafana to visualize metrics by following section on [Setting up Grafana](#grafana)
-* Setup Jaeger analyze tracing as mentioned in section [Setting up Jaeger](#jaeger-server)
-* Setup Elastic Stack only if you are interested in analysing logs by following section on [Setting up Elastic Stack](#elastic-stack)
+* Setup Prometheus for collecting metrics information by following section on [Setting up Prometheus](#setting-up-prometheus)
```
Comment (praneesha) [inline swan-lake/learn/observing-ballerina-code.md:40]:
```suggestion
* Set up Prometheus for collecting metrics information by following the section on [Setting up Prometheus](#setting-up-prometheus)
```

File: swan-lake/learn/observing-ballerina-code.md [hunk 2]
```diff
@@ -37,10 +37,10 @@ Prometheus, etc. You can follow [Docker documentation](https://docs.docker.com/i
...
+* Setup Grafana to visualize metrics by following section on [Setting up Grafana](#setting-up-grafana)
```
Comment (praneesha) [inline swan-lake/learn/observing-ballerina-code.md:41]:
```suggestion
* Set up Grafana to visualize metrics by following the section on [Setting up Grafana](#setting-up-grafana)
```

File: swan-lake/learn/observing-ballerina-code.md [hunk 3]
```diff
@@ -37,10 +37,10 @@ Prometheus, etc. You can follow [Docker documentation](https://docs.docker.com/i
...
+* Setup Jaeger analyze tracing as mentioned in section [Setting up Jaeger](#setting-up-the-jaeger-server)
```
Comment (praneesha) [inline swan-lake/learn/observing-ballerina-code.md:42]:
```suggestion
* Set up Jaeger analyze tracing as mentioned in the section [Setting up Jaeger](#setting-up-the-jaeger-server)
```

File: swan-lake/learn/observing-ballerina-code.md [hunk 4]
```diff
@@ -37,10 +37,10 @@ Prometheus, etc. You can follow [Docker documentation](https://docs.docker.com/i
...
+* Setup Elastic Stack only if you are interested in analysing logs by following section on [Setting up Elastic Stack](#setting-up-elastic-stack)
```
Comment (praneesha) [inline swan-lake/learn/observing-ballerina-code.md:43]:
```suggestion
* Set up Elastic Stack only if you are interested in analysing logs by following the section on [Setting up Elastic Stack](#setting-up-elastic-stack)
```

File: swan-lake/learn/observing-ballerina-code.md [hunk 5]
```diff
@@ -287,15 +286,15 @@ The user can easily identify where the error occurred and information of the err
...
+can be traced using OpenTracing implementations like [Jaeger](http://www.jaegertracing.io/). Jaeger is the default
+tracer of Ballerina.
 
 Semantic tags used by Ballerina also follow the [semantic conventions defined in OpenTracing
 specification.](https://github.com/opentracing/specification/blob/master/semantic_conventions.md)
 
 ### Configuring Advanced Tracing for Ballerina
 
-Tracing can be enabled in Ballerina with `--b7a.observability.enabled=true` flag as mentioned in the [Getting Started](#getting-started) section, as well as configuration option. This section mainly focuses on the configuration options with description and possible values.
+Tracing can be enabled in Ballerina with `--b7a.observability.enabled=true` flag as mentioned in the [Observing a Ballerina Service](#observing-a-ballerina-service) section, as well as configuration option. This section mainly focuses on the configuration options with description and possible values.
```
Comment (praneesha) [inline swan-lake/learn/observing-ballerina-code.md:297]:
```suggestion
Tracing can be enabled in Ballerina with the `--b7a.observability.enabled=true` flag as mentioned in the [Observing a Ballerina Service](#observing-a-ballerina-service) section, as well as a configuration option. This section mainly focuses on the configuration options with the description and possible values.
```

File: swan-lake/learn/observing-ballerina-code.md [hunk 6]
```diff
@@ -310,7 +309,7 @@ The table below provides the descriptions of each configuration option and possi
 Configuration Key | Description | Default Value | Possible Values
 --- | --- | --- | --- 
 b7a.observability.tracing.enabled | Whether tracing is enabled (true) or disabled (false) | false | true or false
-b7a.observability.tracing.name | Tracer name which implements tracer interface. | jaeger | jaeger or zipkin
+b7a.observability.tracing.provider | Tracer name which implements tracer interface. | jaeger | jaeger or if any custom implementation, then name of the tracer.
```
Comment (praneesha) [inline swan-lake/learn/observing-ballerina-code.md:312]:
```suggestion
b7a.observability.tracing.provider | Tracer name which implements tracer interface. | Jaeger | Jaeger or if any custom implementation, the name of the tracer.
```

File: swan-lake/learn/observing-ballerina-code.md [hunk 7]
```diff
@@ -347,125 +346,9 @@ b7a.observability.tracing. jaeger.sampler.param | It is a floating value. Based
...
-b7a.observability.tracing.zipkin. reporter.hostname | Hostname of the Zipkin server | localhost | IP or hostname of the Zipkin server. If it is running on the same node as Ballerina, it can be localhost. 
-b7a.observability.tracing.zipkin. reporter.port | Port of the Zipkin server | 9411 | The port that the Zipkin server is listening to.
-b7a.observability.tracing.zipkin. reporter.api.context | API context of the Zipkin server | `/api/v2/spans` | The API context of the Zipkin API. For V1 API, the context will be `/api/v1/spans`, and for V2 API, the context will be `/api/v2/spans` for default Zipkin server.
-b7a.observability.tracing.zipkin. reporter.api.version | API version of the Zipkin API | v2 | v1 or v2.
-b7a.observability.tracing.zipkin. reporter.compression.enabled | Enable the compression for the spans request | true | true or false.
-
 ### Setting Up the External Systems for Tracing
-Ballerina by default supports Jaerger and Zipkin for distributed tracing. This section focuses on configuring the
-Jaeger and Zipkin with Dockers as a quick installation.
+Ballerina by default supports Jaerger for distributed tracing. This section focuses on configuring the
```
Comment (praneesha) [inline swan-lake/learn/observing-ballerina-code.md:350]:
```suggestion
By default, Ballerina supports Jaeger for distributed tracing. This section focuses on configuring 
```

File: swan-lake/learn/observing-ballerina-code.md [hunk 8]
```diff
@@ -347,125 +346,9 @@ b7a.observability.tracing. jaeger.sampler.param | It is a floating value. Based
...
+Jaeger with Dockers as a quick installation.
```
Comment (praneesha) [inline swan-lake/learn/observing-ballerina-code.md:351]:
```suggestion
Jaeger with Docker as a quick installation.
```

File: swan-lake/learn/observing-ballerina-code.md (hunk 1 above)
Comment (nadundesilva) [reply swan-lake/learn/observing-ballerina-code.md]: Fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2

File: swan-lake/learn/observing-ballerina-code.md (hunk 2 above)
Comment (nadundesilva) [reply swan-lake/learn/observing-ballerina-code.md]: Fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2

File: swan-lake/learn/observing-ballerina-code.md (hunk 3 above)
Comment (nadundesilva) [reply swan-lake/learn/observing-ballerina-code.md]: Fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2

File: swan-lake/learn/observing-ballerina-code.md (hunk 4 above)
Comment (nadundesilva) [reply swan-lake/learn/observing-ballerina-code.md]: Fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2

File: swan-lake/learn/observing-ballerina-code.md (hunk 6 above)
Comment (nadundesilva) [reply swan-lake/learn/observing-ballerina-code.md]: Name "jaeger" should be all in lowercase letter. Other suggestion fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2

File: swan-lake/learn/observing-ballerina-code.md (hunk 7 above)
Comment (nadundesilva) [reply swan-lake/learn/observing-ballerina-code.md]: Fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2

File: swan-lake/learn/observing-ballerina-code.md (hunk 8 above)
Comment (nadundesilva) [reply swan-lake/learn/observing-ballerina-code.md]: Fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2

**Review by praneesha — APPROVED** [review]

**Pull request merged**